packages/lesspass-crypto/index.js

```diff
@@ -1,3 +1,5 @@
+const crypto = require("crypto");
```

Comment:
This one will be problematic;
We need like in lesspass-entropy add some test in the browser introducing karma.

I didn't look at how the browser transform the crypto node module.
But we probably need to have a crypto.browser.js where the encrypt method use `window.crypto`